bin/build/src/build/uberjar.clj

```diff
@@ -96,15 +96,17 @@
   (u/step "Copy resources"
     ;; technically we don't NEED to copy the Clojure source files but it doesn't really hurt anything IMO.
     (doseq [path (all-paths basis)]
-      (u/step (format "Copy %s" path)
-        (b/copy-dir {:target-dir class-dir, :src-dirs [path]})))))
+      (when (not (#{"src" "shared/src" "enterprise/backend/src"} path))
```

Comment:
Are you sure these directories cannot contain resources needed in the jar? E.g., `src/metabase/cmd/resources` looks like it could.

Reply:
good find, but I think it's fine because those resources are used for generating API docstrings which I believe will be published to `metabase.com`.

Anw, this list is taken from this [PR](https://github.com/metabase/metabase/pull/32714/files#diff-e7ed9cc6a5e08eebfeea9b911ba1054d4129a31d079a1537c223f75cc9ae3138).

Reply:
it might actually be better to change this to just copy the `resources` directory explicitly in a single step, instead of looking for all basis paths and then excluding ones that contain source

Reply:
I agree a whitelist would be better. On the other hand, if the goal is to make sure the source code is not there, we could remove the `*.clj{,c,s}` files.